ui/ducks/metamask/metamask.js

```diff
@@ -27,6 +27,7 @@ const initialState = {
   isUnlocked: false,
   isAccountMenuOpen: false,
   isNetworkMenuOpen: false,
+  accountDetailsAddress: '',
```

Comment:
Metamask "duck" is going to be removed in the long run and is meant to be reflective of state keys that are in the background only.  If you are updating this in the background and then pulling the data from the background state (metamask key) you do not need to do this. I have been working towards eliminating these types of "optimistic updates" so unless you're seeing a significant UX issue or delay with just the background state write (which if you are, let me know) then this can be safely removed

Reply:
I've moved this to `appState` as discussed.  I think this is a good temporary measure.